tests/cypress/integration/test.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Basic functionality', function() {
+  before(function() {
+    cy.visit("http://127.0.0.1:8888/lab")
+  })
+  it('Can find launcher icons', function() {
+    cy.get('div.jp-LauncherCard', { timeout: 10000 })
+  })
+  it('Can find main menu at the top', function() {
+    cy.get('div#jp-menu-panel', { timeout: 10000 })
+  })
+  it('Can launch terminal', function() {
+    cy.get('div.jp-LauncherCard[title="Start a new terminal session"]', { timeout: 10000 }).click()
```

Comment:
This doesn't actually verify that it's running, correct?

Reply:
You are right it just verifies that one can click the button to start the terminal but it does not verify that the terminal started and is functional. I think it is fairly easy to issue a few commands and make sure they succeed.